demo/simple_squares/run.py

```diff
@@ -0,0 +1,16 @@
+import gradio as gr
```

Comment:
Suggest naming this folder `blocks_simple_squares` to be consistent with naming of all of the blocks demos.

Reply:
Why are we naming them like this? I'll change for now to be consistent but not a fan of this naming.

Reply:
I think one of the main advantages would be promoting and showing examples for the Blocks release.